tikv/raftstore/node.go

```diff
@@ -197,7 +199,7 @@ func (n *Node) prepareBootstrapCluster(ctx context.Context, engines *Engines, st
 	return PrepareBootstrap(engines, storeID, regionID, peerID)
 }
 
-func (n *Node) BootstrapCluster(ctx context.Context, engines *Engines, firstRegion *metapb.Region) error {
+func (n *Node) BootstrapCluster(ctx context.Context, engines *Engines, firstRegion *metapb.Region) (bool, error) {
```

Comment:
Better name the returned bool value as `newCluster`

Reply:
Use a named return value or return a local variable?